lib/pages/user_profile.dart

```diff
@@ -0,0 +1,185 @@
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: <Widget>[
+        
+        Padding(
+          padding: EdgeInsets.all(32.0),
+          child: Column(
+            children: <Widget>[
+              Container(
+                child: Align(
+                  alignment: Alignment.centerRight,
+                  child: Icon(
+                  Icons.settings,
+                  size: 24.0,
+                  color: DefaultColors.disabledIconColor,
+                ),
+                )
+              ),
+                CircleAvatar(
+                  radius: 50.0,
+                  backgroundImage: NetworkImage('https://www.mobafire.com/images/avatars/jinx-classic.png'),
+                ),
+              SizedBox(
+                height: 16.0,
+              ),
+              Text(
+                'Username',
+                style: TextStyle(
+                  fontSize: 24.0, color: Colors.white),
+              ),
+            ],
+          ),
+        ),
+        Padding(
+          padding: const EdgeInsets.only(left: 32.0, right: 48.0, top: 0.0),
+          child: Row(
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            children: <Widget>[
+              Row(
+                children: <Widget>[
+                  Container(
+                    padding: const EdgeInsets.all(0.0),
+                    width: 32.0,
+                    child:
+                  Icon(
+                    Icons.star,
+                  color: DefaultColors.iconColor
+                  ),
+                  ),
+                  Text(
+                    '100k',
+                    style: TextStyle(
+                  fontSize: 18.0, color: Colors.white),
+                  )
+                ],
+              ),
+              Row(
+                children: <Widget>[
+                  Container(
+                    padding: const EdgeInsets.all(0.0),
+                    width: 32.0,
+                    child:
+                  Icon(
+                    Icons.favorite,
+                  color: DefaultColors.iconColor
+                  ),
+                  ),
+                  Text(
+                    '100k',
+                    style: TextStyle(
+                  fontSize: 18.0, color: Colors.white),
+                  )
+                ],
+              ),
+              Row(
+                children: <Widget>[
+                  Container(
+                    padding: const EdgeInsets.all(0.0),
+                    width: 32.0,
+                    child:
+                  Icon(
+                    Icons.comment,
+                  color: DefaultColors.iconColor
+                  ),
+                  ),
+                  Text(
+                    '100',
+                   style: TextStyle(
+                  fontSize: 18.0, color: Colors.white),
+                  )
+                ],
+              ),
+            ],
+          ),
+        ),
```

Comment:
These rows are the same. Extract them to a method.

Reply:
extracted to a method buildUserInfo